R/utils.r

```diff
@@ -121,3 +121,7 @@ with_dir <- function(new, code) {
   on.exit(setwd(old))
   force(code)
 }
+
+uncranquote <- function(string) {
```

Comment:
`cran_unquote()` is probably better here.

Reply:
Done!